x-pack/plugins/enterprise_search/common/ml_inference_pipeline/index.test.ts

```diff
@@ -97,20 +97,20 @@ describe('getRemoveProcessorForInferenceType lib function', () => {
 });
 
 describe('getSetProcessorForInferenceType lib function', () => {
-  const destinationField = 'dest';
+  const targetField = 'dest';
```

Comment:
I'm taking this opportunity to go with a consistent naming (we call this field "target field" in the UI, not destination field).